UEBPrep/app/views/cards/show.html.erb

```diff
@@ -11,4 +11,8 @@
     </div>
   </div>
   <%= render @card.content %>
-</div>
\ No newline at end of file
+</div>
+
+<%unless @current_user.nil? %>
+    <%= render 'links' %>
```

Comment:
In the future we can scope this to being shown only to people that have rights to edit or view or create cards.